clone_linkedin/post/views.py

```diff
@@ -53,3 +57,32 @@ def update(self, request, pk=None):
     def destroy(self, request, pk=None):
         self.get_object().delete()
         return Response()
+        
+    # GET, POST, PUT, DELETE /posts/:id/reaction/
+    @action(detail=True, methods=['GET', 'POST', 'PUT', 'DELETE'], url_path='reaction')
+    def reaction(self, request, pk=None):
+        post = self.get_object()
+        data = request.data.copy()
+        if self.request.method == 'GET':
+            reaction = self.get_object().postReactions.all()
+            return Response(self.get_serializer(reaction, many=True).data)
+        elif self.request.method == 'POST':
+            user = User.objects.get(id=1)          # Needs to be fixed after login implemented.
+            if self.get_object().postReactions.filter(user=user).exists():
+                return Response({"error": "Reaction already made."}, status=status.HTTP_400_BAD_REQUEST)
+            data['user_id'] = 1
+            data['post_id'] = post.id 
+            serializer = self.get_serializer(data=data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        elif self.request.method == 'PUT':
+            serializer = self.get_serializer(data=data) 
+            serializer.is_valid(raise_exception=True) 
+            serializer.update(serializer.validated_data)
+            return Response(serializer.data) 
```

Comment:
`PUT` 부분 고쳐야 할 것 같아요~
```suggestion
        elif self.request.method == 'PUT':
            user = User.objects.get(id=1)    # Needs to be fixed after login implemented
            reaction = get_object_or_404(post.postReactions, user=user)
            serializer = self.get_serializer(reaction, data=data, partial=True)
            serializer.is_valid(raise_exception=True) 
            serializer.update(reaction, serializer.validated_data)
            return Response(serializer.data) 
```
그리고 이러면 `PUT /posts/:id/reaction/` request 보낼 때 body에 `post_id`나 `user_id` key 있으면 에러 나니까 **PostReactionSerializer**에 `update` method 추가해주세요